backend/fleet_management/serializers.py

```diff
@@ -1,9 +1,26 @@
 from rest_framework import serializers
 
-from .models import User
+from .models import User, Route
 
 
 class UserSerializer(serializers.ModelSerializer):
     class Meta:
         model = User
         fields = ('id', 'username')
+
+
+class RouteSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Route
+        fields = (
+            'car',
```

Comment:
Can you remind how will i.e. car be serialized?

Reply:
by id?

Reply:
I think we will more than id to show car data. Take a look at nesting serializers: https://www.django-rest-framework.org/api-guide/relations/#example